pkgs/build-support/node/build-npm-package/default.nix

```diff
@@ -1,4 +1,11 @@
-{ lib, stdenv, fetchNpmDeps, buildPackages, nodejs } @ topLevelArgs:
+{ lib
+, stdenv
+, fetchNpmDeps
+, buildPackages
+, nodejs
+, cctools
```

Comment:
Shouldn't we just be accepting `darwin` here?

I was under the impression splitting up and inheriting darwin stuff in all-packages.nix was an anti-pattern that we've started discouraging, but I may be misremebering (I'm also on mobile or I'd look it up)

Reply:
I wasn't aware of that. I changed it to accept `darwin` instead.